package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "sphere-product-type-json-generator",
   "description": "A command line tool for generating ProductTypes JSON representations from CSV templates.",
-  "version": "0.2.8",
+  "version": "0.3.0",
```

Comment:
Don't bump the version now. This is usually done automatically by the release task.